docs/pages/blog/introducing-pigment-css.md

```diff
@@ -0,0 +1,114 @@
+---
+title: 'Introducing Pigment CSS: the next generation of CSS-in-JS for Material UI'
```

Comment:
```suggestion
title: 'Introducing Pigment CSS: the next generation of CSS-in-JS library'
```

Was it intentional the part "for Mateiral UI"? People may read it as only for Material UI.

Reply:
The title is too long. I just learned that we have this error in the CI.

<img width="1207" alt="image" src="https://github.com/mui/material-ui/assets/18292247/335aa956-7ea9-4072-9de9-f4addbc55bd2">